shared/src/test/scala/pl/metastack/metatime/DateTimeTest.scala

```diff
@@ -24,9 +26,16 @@ class DateTimeTest extends FunSuite {
     assert(DateTime(2016,1,1,1,1,1,1) + Day(1) == DateTime(2016, 1, 2, 1, 1, 1, 1))
   }
 
-  ignore("now") {
-    assert(DateTime.now() == ???)
-  }
+  test("now") {
+      val scheduler: Scheduler = Platform.DefaultScheduler
+      val dt = DateTime.now()
+      val dtAfter = (dt + Second(5)).asInstanceOf[DateTime]
+      val p = Promise[Boolean]()
+      scheduler.at(dtAfter) {
+        p.success(Math.abs(DateTime.now().unix().value - dt.unix().value) < 10000)
```

Comment:
It should be possible to write `p.success(DateTime.now() - dt < 10000)` here.

`abs(x)` is redundant because `dt` is expected to be `<=` `DateTime.now()`.